data-overview/english.md

```diff
@@ -71,27 +73,28 @@ My Trips allows you to get a history of your cars trips. You can see where you h
 | Parameter | Purpose of data processing | Storage |
 |-----------|-----------------------|------------|
 | Car GPS position | So the user can see the car's route on a map | Saved until user requests deletion of trip |
-| Direction of car | So the user can see the car's route on a map, used to optimize route viewing | Saved until user requests deletion of trip |
-| Car fuel level | So the user can see the car's estimated fuel consumption | 60 days historical data. Calculated fuel consumption is saved until user requests deletion of trip |
-| Car mileage | So the user can see the number of kilometers traveled for each trip | 60 days historical data. Calculated distance on trip is saved until user requests deletion of trip |
-| Drive in time | So the user can see the duration of the trip | Saved until user requests deletion of trip |
-| Road toll* | So the user can get an estimate of toll cost per trip in the trip export | Saved until user requests deletion of trip |
+| Direction of car | So the user can see the car's route on a map, used to optimize route viewing | Saved until user requests deletion |
+| Car fuel consumption | So the user can see the car's estimated fuel consumption | Saved until user requests deletion |
+| Car refuel events | So the user can see when the car was refueled and how much | Saved until user requests deletion |
+| Car mileage | So the user can see the number of kilometers traveled for each trip | 60 days historical data. Calculated distance on trip is saved until user requests deletion |
+| Drive in time | So the user can see the duration of the trip | Saved until user requests deletion |
```

Comment:
Perhaps add that we also infer the IdleTime from this or reformulate so we collect data about the time usage of the trip.

Reply:
I've just rephrased to "Time consumption" and added idle time in the description.